polaris-core/src/main/java/org/apache/polaris/core/config/BehaviorChangeConfiguration.java

```diff
@@ -37,9 +37,10 @@ protected BehaviorChangeConfiguration(
     super(key, description, defaultValue, catalogConfig);
   }
 
+  @BehaviorChangeConfigurationSince("1.0.0")
```

Comment:
We need to coordinate setting this value with the new 0.10.0 release proposal: https://lists.apache.org/thread/8kc7xq3lp6pcc2w3jon3fv8qfxj3v9lk

Reply:
We need to fix the `versions.txt` to point to the actual version of the code, I think. These values should be set to whatever is there. Currently, that's 1.0.0